src/MARIGOLD/iate.py

```diff
@@ -0,0 +1,364 @@
+    
+    ############################################################################################################################
+    #                                                                                                                          #
+    #                                                      COEFFICIENTS                                                        #
+    #                                                                                                                          #
+    ############################################################################################################################
+    # coeff.m
+    
+    C_WE            = 0.002                                     # Wake entrainment coefficient, Talley
+    C_RC            = 0.004                                     # Random collision coefficient
+    C_TI            = 0.085                                     # Turbulent impact coefficient
+    
+    alpha_max       = 0.75
+    C               = 3                                         # What is this
+    We_cr           = 6                                         # Weber number criterion
+
+    acrit_flag      = 0
+    acrit           = 0.13
+    # C_WE            = 0.004                                     # Wake entrainment coefficient, Worosz
+
+    C0              = 1.20                                      # Drift Flux Model
+
+    # Yadav
+    if theta == 90 and elbow == False:
+        C_WE        = 0.002
+        C_RC        = 0.004
+        C_TI        = 0.085
+
+        alpha_max   = 0.75
+        C           = 3
+        We_cr       = 6
+
+    elif theta == 0 and elbow == False:
+        C_RC        = 0.003
+        C_TI        = 0.014
+
+        We_cr       = 5
+
+    elif elbow == True:
+        C_RC        = 0.012
+        C_TI        = 0.085
+
+        We_cr       = 6
+        
+        slip        = 0.91                                      # Slip at port P4 (<<vg>>/<<vf>>)
+        Const1      = 0.05                                      # Constant in the correlation for velocity
+    
+    ############################################################################################################################
+    #                                                                                                                          #
+    #                                                       COVARIANCE                                                         #
+    #                                                                                                                          #
+    ############################################################################################################################
+    # COV.m
+
+    # Yadav
+    # Temporary, Yadav implemented as a bunch of arrays. There must be a better way to do this.
+    # calc_void_cov in Condition.py may be useful
+    if theta == 90 and elbow == False:
```

Comment:
Drew had his method, that I doubt even made it out of Excel. We probably should implement that